download_clients/pivnet_client.go

```diff
@@ -95,6 +97,11 @@ func (p *pivnetClient) GetLatestProductFile(slug, version, glob string) (FileArt
 		return nil, fmt.Errorf("could not fetch the release for %s %s: %s", slug, version, err)
 	}
 
+	err = p.downloader.AcceptEULA(slug, release.ID)
+	if err != nil {
+		return nil, fmt.Errorf("could not accept EULA for download product file %s at version %s: %s", slug, version, err)
```

Comment:
nit: what are your thoughts on wrapping the error with the `%w` verb? something like this
```
return nil, fmt.Errorf("could not accept EULA for download product file %s at version %s: %w", slug, version, err)

```